### PR TITLE
[CM T1] jna: Improve JNA documentation

### PR DESCRIPTION
**Pull request by bhampl**
## Basics

- [x] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [x] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you

## Who did what

@bhampl improved the JNA documentation

### CODE REVIEW

**Review by kodebach — COMMENTED** [review]
Good work, please rebase with `master` to fix the build server problems.

File: src/bindings/jna/README.md [hunk 1]
```diff
@@ -46,7 +48,7 @@ libelektra.jar), e.g.:
 export CLASSPATH=".:/usr/share/java/libelektra.jar:/usr/share/java/jna.jar"
 ```
 
-Then you can compile and run [HelloElektra](HelloElektra.java):
+Then you can compile and run [HelloElektra](hello/src/main/java/HelloElektra.java):
 
 ```sh
 javac HelloElektra.java && java HelloElektra
```
Comment (markus2330) [inline src/bindings/jna/README.md:54]: Did this work for you?

**Review by markus2330 — COMMENTED** [review]
Thank you for the PR!

File: src/bindings/jna/README.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ You can run the `HelloElekra` example by executing:
 ./gradlew hello:run
 ```
 
+If this fails, make sure you have at least Java version 11 or higher installed.
```
[the remark this answers is not in the document]
Comment (bhampl) [reply src/bindings/jna/README.md:20]: Yes. But I forgot to mention that you have to set the `JAVA_HOME` environment variable correctly, so I will add that.

File: src/bindings/jna/README.md (hunk 1 above)
Comment (bhampl) [reply src/bindings/jna/README.md:54]: Yes, that works for me (provided I have set the `CLASSPATH` correctly as mentioned above).

**Pull request merged**